formulations/+MPCT/compute_MPCT_EADMM_ingredients.m

```diff
@@ -54,33 +53,33 @@
         try
             LBx = controller.sys.LBx;
         catch
-            LBx = -options.inf_bound*ones(n, 1);
+            LBx = -opt.inf_value*ones(n, 1);
         end
         try
             UBx = controller.sys.UBx;
         catch
-            UBx = options.inf_bound*ones(n, 1);
+            UBx = opt.inf_value*ones(n, 1);
         end
         try
             LBu = controller.sys.LBu;
         catch
-            LBu = -options.inf_bound*ones(m, 1);
+            LBu = -opt.inf_value*ones(m, 1);
         end
         try
             UBu = controller.sys.UBu;
         catch
-            UBu = options.inf_bound*ones(m, 1);
+            UBu = opt.inf_value*ones(m, 1);
         end
     end
-    
+
     %% Extract or compute rho
-    if isfield(options, 'rho')
-        options.rho_base = options.rho;
-        options.rho_mult = 1;
+    if isfield(opt.solver, 'rho')
+        opt.solver.rho_base = options.rho;
```

Comment:
In this line, we should have "opt.solver.rho_base = opt.solver.rho;"

Furthermore, that line would generate an error saying that we cannot change opt.solver.rho_base because it is read-only. In order to fix this, you have to set to "public" the options. I leave another comment in the correspondent file.